gtsam_unstable/slam/PoseToPointFactor.h

```diff
@@ -45,8 +45,9 @@ class PoseToPointFactor : public NoiseModelFactor2<POSE, POINT> {
   /** implement functions needed for Testable */
 
   /** print */
-  virtual void print(const std::string& s, const KeyFormatter& keyFormatter =
-                                               DefaultKeyFormatter) const {
+  virtual void print(
```

Comment:
I think we should not have virtual *and* override. One or the other. Surprised that does not trigger a warning (check clang)

Reply:
good catch, my mistake! I just fixed it